testsuite/MDAnalysisTests/converters/test_parmed_parser.py

```diff
@@ -150,32 +215,37 @@ def test_angles_atom_counts(self, universe):
         assert len(universe.atoms[[0]].angles), 9
         assert len(universe.atoms[[42]].angles), 2
 
-    @pytest.mark.parametrize('value', (
-        (1, 0, 2),
-        (1, 0, 3),
-        (1, 0, 4),
-        ))
+    @pytest.mark.parametrize(
+        "value",
+        (
+            (1, 0, 2),
+            (1, 0, 3),
+            (1, 0, 4),
+        ),
+    )
     def test_angles_identity(self, top, value):
         vals = top.angles.values
         assert value in vals or value[::-1] in vals
 
     def test_dihedrals_atom_counts(self, universe):
         assert len(universe.atoms[[0]].dihedrals) == 14
 
-    @pytest.mark.parametrize('value', (
-        (0, 4, 6, 7),
-        (0, 4, 6, 8),
-        (0, 4, 6, 9),
-        (0, 4, 17, 18),
-        ))
+    @pytest.mark.parametrize(
+        "value",
+        (
+            (0, 4, 6, 7),
+            (0, 4, 6, 8),
+            (0, 4, 6, 9),
+            (0, 4, 17, 18),
+        ),
+    )
     def test_dihedrals_identity(self, top, value):
         vals = top.dihedrals.values
         assert value in vals or value[::-1] in vals
 
-    @pytest.mark.parametrize('value', (
-        (17, 19, 21, 41, 43),
-        (60, 62, 64, 79, 81)
-    ))
+    @pytest.mark.parametrize(
+        "value", ((17, 19, 21, 41, 43), (60, 62, 64, 79, 81))
```

Comment:
```suggestion
        "value", ((17, 19, 21, 41, 43), (60, 62, 64, 79, 81),)
```

Reply:
I think this one is alright since it fits in one line.